packages/core/src/base/typography.scss

```diff
@@ -217,7 +217,7 @@ Style guide: style.typography.responsive
 */
 
 /*
-Customizing type sizes
```

Comment:
Just curious, whats the reasoning for this change? Do other pages with similar docs also call it "Customization"?

Reply:
Yeah there was a mix between `theming` and `customization` I'm open to suggestions too. 
This is where SASS variables will be displayed to help with styling